Makefile

```diff
@@ -58,6 +58,44 @@ else
 endif
 endif
 
+.PHONY: help
+help:
+	@echo "Available targets:"
+	@echo "  aks.kubeconfig        - Get admin AKS kubeconfig"
+	@echo "  aro                   - Build the aro binary"
+	@echo "  build                 - Build binaries"
+	@echo "  build-all             - Build all binaries"
```

Comment:
A bit more information about the difference in what binaries are built with the build vs build-all targets would be helpful here.